src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/222185804-f75afd21-1d48-4bfb-b03a-96567cd0f1dc.png)
Better specify the exact tags you want to change. Also, you can set the font-family for the whole page and inherit where needed

Reply:
Thank you! Links changed, copied them from the old task by accident)